ibis/pandas/core.py

```diff
@@ -85,13 +85,35 @@
 :class:`~ibis.expr.operations.ScalarParameter`, in which case the ``scope``
 passed to ``post_execute`` would be the bound values passed in at the time the
 ``execute`` method was called.
+
+
+Scope
+-------------------
+Scope is used across the execution phases, it iss a map that maps Ibis
+operators to actual data. It is used to cache data for calculated ops. It is
+an optimization to reused executed results.
+
+With time context included, the key is op associated with each expression;
+And scope value is another key-value map:
+- value: pd.DataFrame or pd.Series that is the result of executing key op
+- timecontext: of type TimeContext, the time context associated with the data
+stored in value
+
+Note that the idea is: data should be determined by both op and timecontext.
```

Comment:
I think this is an implementation deal of scope and should be explained here. From the point of execution, it doesn't care whether time context is stored as part of the dict key or as a part of the dict value, as long as the scope API remains the same.

Reply:
Make sense, I will add it here.